examples/src/main/scala/org/apache/spark/examples/ml/DataFrameExample.scala

```diff
@@ -24,7 +24,8 @@ import com.google.common.io.Files
 import scopt.OptionParser
 
 import org.apache.spark.examples.mllib.AbstractParams
-import org.apache.spark.mllib.linalg.Vector
+import org.apache.spark.ml.linalg.Vector
+import org.apache.spark.mllib.linalg.VectorImplicits._
```

Comment:
`VectorImplicits` shouldn't appear in example code. I created https://issues.apache.org/jira/browse/SPARK-15363 to track it.